site/app/libraries/database/AbstractDatabase.php

```diff
@@ -143,6 +143,16 @@ public function isConnected() {
     public function query($query, $parameters = array()) {
         try {
             $this->query_count++;
+            foreach ($parameters as &$parameter) {
+                if (gettype($parameter) === "boolean") {
+                    $parameter = $this->convertBoolean($parameter);
+                }
+                elseif (gettype($parameter) === "object") {
+                    if (get_class($parameter) === "DateTime") {
+                        $parameter = $parameter->format("Y-m-d H:i:sO") . " " . $parameter->getTimezone()->getName();
```

Comment:
Looking at this, I believe we can just rid of the concatenation (so just have `$parameter => $parameter->format("Y-m-d H:i:sO");`) as the `O` provides us the UTC offset and I think the DB is just stripping off the concatenation anyway.